README.md

```diff
@@ -1,2 +1,2 @@
 # 2018-Java
-Repository for the 2018 build. Java was used this year. 
+Repository for FRC team 4418's 2018 software build. Java was used this year. 
```

Comment:
Description change verified.

Reply:
However, someone other than me (Will) has to review this change and approve it with the way I set up the repository.

Reply:
That's cool!